app/views/taxons/sections/_news_and_communications.html.erb

```diff
@@ -1 +1,44 @@
-<%= render partial: 'document_list_with_grid', locals: { documents: section[:documents] } %>
+<%
+  featured_item = section[:documents].shift(section[:promoted_content_count]).first
+  featured_item_layout_class = "taxon-page__featured-item--single" if section[:documents].empty?
+%>
+
+<div class="grid-row">
+  <div class="taxon-page__featured-item column-one-third <%= featured_item_layout_class %>">
+    <img class="taxon-page__featured-image" src="<%= featured_item[:image][:url] %>" alt="<%= featured_item[:image][:alt] %>">
+    <div class="taxon-page__featured-text">
+      <a class="taxon-page__featured-link" href="<%= featured_item[:link][:path] %>"><%= featured_item[:link][:text] %></a>
+      <div class="taxon-page__featured-metadata-wrapper">
+        <% featured_item[:metadata].each do |metadata_key, metadata_value| %>
+          <% if metadata_key.to_s.eql?("public_updated_at") %>
+            <time class="taxon-page__featured-metadata" datetime="<%= metadata_value.iso8601 %>">
+              <%= l(metadata_value, format: '%e %B %Y') %>
+            </time>
+          <% else %>
+            <p class="taxon-page__featured-metadata"><%= metadata_value %></p>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+
+  <% if section[:documents].empty? %>
+      <%= link_to(
+        section[:see_more_link][:text],
+        section[:see_more_link][:url], class: "taxon-page__featured-see-more"
+      )%>
+    </div>
```

Comment:
I was hesitant about doing it, but at the time decided to put it above readability of code. I've implemented a different solution which now uses 2 if statements rather than 1 if/elsif. The code is longer, but I think it makes it easier to read?